release.sh

```diff
@@ -62,3 +66,16 @@ gcutil push fe tmp/$CHANNEL /var/www/install.sandstorm.io
 
 gcutil ssh smalldemo sudo service sandstorm update
 gcutil ssh alpha sudo service sandstorm update dev
+
+echo "**** Tagging this commit ****"
```

Comment:
Nitpick: IMO this section should execute before "Pushing build", so that a random problem with production machines doesn't mean we lose the tag.